chainbench/profile/evm/heavy.py

```diff
@@ -10,26 +12,129 @@ class EthereumHeavyProfile(EVMBenchUser):
     wait_time = constant_pacing(10)
 
     @task
-    def trace_transaction_task(self):
+    def debug_trace_transaction_task(self):
         self.make_call(
-            name="trace_transaction",
+            name="debug_trace_transaction",
             method="debug_traceTransaction",
-            params=[],
+            params=self._trace_transaction_by_hash_params_factory(),
         ),
 
     @task
-    def block_task(self):
+    def trace_block_task(self):
         self.make_call(
-            name="block",
+            name="trace_block",
             method="trace_block",
             params=self._block_by_number_params_factory(),
         ),
 
     @tag("get-logs")
     @task
-    def get_logs_task(self):
+    def eth_get_logs_task(self):
         self.make_call(
-            name="get_logs",
+            name="eth_get_logs",
             method="eth_getLogs",
             params=self._get_logs_params_factory(),
         ),
+
+    @task
+    def eth_get_blocks_receipts_task(self):
+        self.make_call(
+            name="eth_get_block_receipts",
+            method="eth_getBlockReceipts",
```

Comment:
It looks like the `name` pretty much repeats the `method` everywhere, why not making `name` optional and using `method` when it's missing?

Reply:
Yeah you're right, I'm not sure why Egor made it that way, but we could change/enhance it.